packages/workflow-core/src/lib/plugins/common-plugin/iterative-plugin.ts

```diff
@@ -18,40 +18,54 @@ export class IterativePlugin {
     this.action = pluginParams.action;
     this.successAction = pluginParams.successAction;
     this.errorAction = pluginParams.errorAction;
+    console.log(`Constructed IterativePlugin with params: ${JSON.stringify(pluginParams)}`);
```

Comment:
Let's add a tech debt task to add a debug mode which can be default false in non-dev. Debug mode false won't log or maybe not all logs.